python/GafferUI/ColorChooser.py

```diff
@@ -207,6 +348,12 @@ def colorChangedSignal( self ) :
 
 		return self.__colorChangedSignal
 
+	def setColorFieldVerticalSizePolicy( self, policy ) :
```

Comment:
This exposes Qt types in the public API, which is a no-no. `set` methods are also always paired with `get` methods.

Reply:
Right, thanks for pointing that out. I removed it in my latest push since the widget is a fixed size while we test it.